common/src/main/java/tachyon/Constants.java

```diff
@@ -101,6 +101,7 @@
   public static final String IN_TEST_MODE = "tachyon.test.mode";
   public static final String NETWORK_HOST_RESOLUTION_TIMEOUT_MS =
       "tachyon.network.host.resolution.timeout.ms";
+  public static final String THRIFT_STOP_TIMEOUT_SECONDS = "tachyon.thrift.stop.timeout.seconds";
```

Comment:
Is this worth documenting in `Configuration-Settings.md`? I don't think users would care about it, so I'm leaving it out for now.